Makefile

```diff
@@ -61,7 +65,14 @@ develop:
 	$(PIP) install -e .
 
 install: clean
-	$(PIP) install -e ."[api, datadog, prometheus, elasticsearch, pubsub, cloud_monitoring, bigquery, dev]"
+ifeq ($(OS),Darwin)
+	@echo "running on MacOs, installing snappy headers with brew"
+	brew install snappy
+else
+	@echo "running on Linux, assuming Ubuntu, installing snappy headers with apt"
+	sudo apt install libsnappy-dev -y || apt install libsnappy-dev -y
+endif
+	$(PIP) install -e ."[api, datadog, prometheus, elasticsearch, pubsub, cloud_monitoring, bigquery, dev, prometheus_remote_write]"
```

Comment:
I'd prefer to keep this clean of additional dependencies, can we just document those steps in `docs/providers/prometheus_remote_write.md` "7u8i9uojkl,;. æ÷{
}